docs/data-sources/telephony.md

```diff
@@ -0,0 +1,37 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "dcloud_licenses Data Source - terraform-provider-dcloud"
+subcategory: ""
+description: |-
+  All the licenses currently in a given topology
```

Comment:
Need to regenerate the doc as it's referencing LIcenses.

Reply:
Fixed.